cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -296,6 +296,7 @@ private HttpServletRequest prepareRequest(HttpSession session, String userTaskId
     EasyMock.expect(request.getSession(false)).andReturn(session).anyTimes();
     EasyMock.expect(request.getMethod()).andReturn(GET_METHOD).anyTimes();
     EasyMock.expect(request.getRequestURI()).andReturn(resource).anyTimes();
+    EasyMock.expect(request.getPathInfo()).andReturn(resource).anyTimes();
```

Comment:
(Not introduced in this PR) Can we add `verify` for mocks? (please see earlier similar comment)

Reply:
Same comment as above

Reply:
Sure, we can address them in a separate PR -- can we create an issue about them?